### DISCUSSION TITLE
login.cl is not created

### DISCUSSION THREAD

**Discussion by s-a-ki** [iraf-community/iraf#336] [Q&A] [answered]
I have installed iraf-2.17.1 based on the site.
However, when I run ’mkiraf’ , only the uparm directory is created, but 'login.cl' is not created.

Is there any possible cause for this?

platform:ubuntu22.04LST

**Comment by olebole** [comment, marked as answer]
In 2.17.1, a copy of an unchanged `login.cl` is no longer needed, therefore it is not created by default. You can still create one by running `mkiraf -c`.
